heartbeat/oracle

```diff
@@ -440,6 +442,12 @@ check_mon_user() {
 			return 1
 		fi
 	fi
+
+	if [ -z "$OCF_RESKEY_monpassword" ]; then
+		ocf_exit_reason "Please explicitly set a password for $MONUSR oracle user"
+		return $OCF_ERR_CONFIGURED
```

Comment:
Only thing is, this return code won't be kept/interpreted when check_mon_user() returns.

Reply:
Nice catch. Seems like using exit instead of return would be the easiest way to solve that.